app/src/main/java/org/simple/clinic/scanid/ScanSimpleIdScreen.kt

```diff
@@ -0,0 +1,67 @@
+package org.simple.clinic.scanid
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.util.AttributeSet
+import android.widget.Toast
+import androidx.appcompat.widget.Toolbar
+import androidx.constraintlayout.widget.ConstraintLayout
+import com.jakewharton.rxbinding2.view.RxView
+import io.reactivex.Observable
+import io.reactivex.android.schedulers.AndroidSchedulers.mainThread
+import io.reactivex.schedulers.Schedulers.io
+import kotterknife.bindView
+import org.simple.clinic.R
+import org.simple.clinic.activity.TheActivity
+import org.simple.clinic.router.screen.ScreenRouter
+import org.simple.clinic.search.PatientSearchScreenKey
+import org.simple.clinic.widgets.ScreenDestroyed
+import org.simple.clinic.widgets.UiEvent
+import org.simple.clinic.widgets.hideKeyboard
+import org.simple.clinic.widgets.qrcodescanner.QrCodeScannerView
+import javax.inject.Inject
+
+class ScanSimpleIdScreen(context: Context, attrs: AttributeSet) : ConstraintLayout(context, attrs) {
```

Comment:
The "record every patient…" text on home screen has very little bottom space on a 1080p device. Can you get this reviewed by Akshay?

![screenshot_1550568997](https://user-images.githubusercontent.com/2387680/53004899-1bd1a000-3458-11e9-8159-8fd59afc189f.png)

Reply:
The text has been removed in a future design.